src/analysis_file.py

```diff
@@ -86,6 +80,8 @@ def generate(user, data, csv_by_message_output_path, csv_by_individual_output_pa
             td.append_data(analysis_dict,
                            Metadata(user, Metadata.get_call_location(), TimeUtils.utc_now_as_iso_string()))
 
+        export_keys = list(fold_strategies.keys())
```

Comment:
This is why fold_strategies was set to an OrderedDict() here https://github.com/AfricasVoices/Project-OCHA/pull/62#discussion_r349038215